.changeset/tender-jobs-double.md

```diff
@@ -0,0 +1,5 @@
+---
+'imagetools-core': major
```

Comment:
why wouldn't this just be a bug fix? the API doesn't change at all
```suggestion
'imagetools-core': patch
```

Reply:
It's a breaking change in as much as it will change the image IDs produced and these image IDs may be being used in various ways by consumers of the package. For instance this change would break some snapshot tests in my project where image IDs in the snapshot would no longer match and I would need to update the snapshots to contain the new IDs.

Reply:
given that it only affects tests and not actual users I'm going to go ahead and say it's not worth a major version bump. if we're too strict we're going to end up on version 100 before long